### ISSUE TITLE
Original PlayStation 3 Controller didn't work!

### ISSUE THREAD

**Issue by spielvan** [xerpi/fakemote#1]
I made the installation following step by step, but none of my 3 original playstation 3 controls worked, remembering that I use them perfectly with nintendont and retroarch on the wii, but with fakemote the lights are blinking and not fixed, very strange , will it still only work with playstation 4 control? Another thing, how is the wii remote cursor using the playstation 3 control, to move it use the L3?

**Comment by Invictaz** [comment]
@spielvan 

-L3 move for nunchuck camera can only be done if you recompile the project with altered code
-Full custom mapper is coming later
-Blinking leds is known

**Comment by eadmaster** [comment]
> but with fakemote the lights are blinking and not fixed

in my case when i connect the controller to my wii the leds do not start blinking at all (tried with both usb ports).
Am i missing something?

**Comment by spielvan** [comment]
> > but with fakemote the lights are blinking and not fixed
> 
> in my case when i connect the controller to my wii the leds do not start blinking at all (tried with both usb ports and 2 different controllers). Am i missing something?

It needs to be original ps3/ps4 controls and they have the same Vid and Pid that you have in the fakemote compatibility table, to use the two USB ports you need to install it on ios58.

**Comment by eadmaster** [comment]
yes, the vid and pids matches, one conttoller i've tried is the original DS3.

Which usb port is supposed to work on ios57, upper or lower?

I'm booting games from the internal SD via Wiiflow. Any custom setting is reaquired in the usb loader?

**Comment by eadmaster** [comment]
ok, figured out you must plug the controller in the UPPER USB port while the system is turned on or it won't be recognized.
